psi4/share/psi4/scripts/test_threading.py

```diff
@@ -187,22 +187,27 @@ def print_math_ldd(args):
     modcore = mod.__file__[:-11] + sharedlibrary
 
     if sys.platform.startswith('linux'):
-        cmd = """ldd -v {} | grep -e ':' -e 'mkl' -e 'openblas' -e 'iomp5' -e 'gomp'""".format(modcore)
-        print('Running {} ...'.format(cmd))
-        subprocess.call(cmd, shell=True)
-        lddout = subprocess.getoutput(cmd)
-        report = {'mkl': lddout.count('libmkl'),
-                  'iomp5': lddout.count('libiomp5'),
-                  'openblas': lddout.count('libopenblas'),
-                  'gomp': lddout.count('libgomp')}
-        print(report)
-        report = {k : bool(v) for k, v in report.items()}
-        okmkl = report['mkl'] and report['iomp5'] and not report['openblas'] and not report['gomp']
-        okopenblas = not report['mkl'] and not report['iomp5'] and report['openblas'] and report['gomp']
-        if args.passfail:
-            assert okmkl != okopenblas
+        lddish = 'ldd -v'
+    elif sys.platform.startswith('darwin'):
+        lddish = 'otool -L'
     else:
-        print('Not available w/o `ldd`')
+        print('Not available w/o `ldd` or `otool`')
+        return True
+
+    cmd = """{} {} | grep -e ':' -e 'mkl' -e 'openblas' -e 'iomp5' -e 'gomp'""".format(lddish, modcore)
+    print('Running {} ...'.format(cmd))
+    subprocess.call(cmd, shell=True)
+    lddout = subprocess.getoutput(cmd)
```

Comment:
isn't the `call` command redundant here?

Reply:
Not that performance matters - just curious

Reply:
Did a little googling on this just because I'm not familiar with `getoutput` (I use `communicate` normally).  Looks like it's new to py3, so just a heads-up that this could break our soon-to-be deprecated py27 support, if I'm right

Reply:
Hmm, shouldn't this just `lddout = subprocess.check_output(cmd, shell=True)`?

Reply:
I think so, which is why I commented initially

Reply:
The pipe complicates matters a great deal. It didn't like the substitution.

Guess I'll know why py27 fails, then. Thanks for the warning.